update benchmarks CLI

PierreGtch opened the pull request:
Hi, what would you say about using [Fire](https://github.com/google/python-fire) to launch benchmarks from a CLI instead of the `run.py` script?

bruAristimunha commented:
I like the idea, maybe it is help to better control the experiments. Can you update the poetry to include fire?

PierreGtch commented:
> I like the idea, maybe it is help to better control the experiments. Can you update the poetry to include fire?

@bruAristimunha I removed the `run.py` file and updated the `run_docker.py` file. Can you check this docker file? I'm not sure how it is used
